src/config/keys.ts

```diff
@@ -0,0 +1,20 @@
+import dev from './dev';
+import prod from './prod';
+import test from './tst';
+
+const env = process.env.NODE_ENV?.trim();
+let keys: any;
```

Comment:
1- Make an interface for `keys` for type-checking.
2- rename the file and exported object from `keys` to `config`

Reply:
check the changes